frontend/src/reducers/accessRequestReducer.tsx

```diff
@@ -0,0 +1,28 @@
+import * as actionTypes from 'constants/actionTypes';
+import { IAccessRequest, IStoreAccessRequestsAction } from 'actions/adminActions';
+
+export interface IAccessRequestState {
+  pagedAccessRequests: IAccessRequest;
+}
+
+const initialState: IAccessRequestState = {
+  pagedAccessRequests: { page: 0, total: 0, quantity: 0, items: [] },
+};
+
+const accessRequestReducer = (state = initialState, action: IStoreAccessRequestsAction) => {
+  switch (action.type) {
+    case actionTypes.STORE_ACCESS_REQUESTS:
+      console.log('storing access request');
```

Comment:
/nit console.logs are fine for debugging but we should try to take them out out public PRs. We might need to automate this as myself keep forgetting to remove them too!